contracts/colony/ColonyFunding.sol

```diff
@@ -106,6 +107,57 @@ contract ColonyFunding is
     emit ColonyFundsClaimed(msgSender(), _token, feeToPay, remainder);
   }
 
+  function claimDomainFunds(address _token, uint256 _domainId) public stoppable {
+    require(domainExists(_domainId), "colony-funding-domain-does-not-exist");
+    address domainTokenReceiverAddress = IColonyNetwork(colonyNetworkAddress)
+      .checkDomainTokenReceiverDeployed(_domainId);
+    uint256 fundingPotId = domains[_domainId].fundingPotId;
+    // It's deployed, so check current balance of pot
+    uint256 balanceBefore = getFundingPotBalance(fundingPotId, _token);
+
+    uint256 claimAmount;
+    uint256 thisBalanceBefore;
+
+    if (_token == address(0x0)) {
```

Comment:
Why not just have `transferToColony` return the token value? That would save you this before/after accounting.

Reply:
To be honest, the asserts post-calling `transferToColony` was mostly just me being careful / checking how things were going during the implementation - it's not like we do it in `claimColonyFunds`.